js/index.js

```diff
@@ -0,0 +1,31 @@
+const btn = document.querySelector('.addBtn');
+const bookContainer = document.querySelector('.bookContainer');
+const books = JSON.parse(localStorage.getItem('books')) || [];
+function onAddElement(e) {
+  e.preventDefault();
+  const title = document.getElementById('insertTitle').value;
+  const author = document.getElementById('insertAuthor').value;
+  bookContainer.innerHTML += `
+        <section>   
+        <p>${title}</p>
+            <p>${author}</p>
+            <button class="deleteBtn" >Remove</button>
+            <hr>
+            </section>
+        `;
+  books.push({ title, author });
+  localStorage.setItem('books', JSON.stringify(books));
+  console.log(books);
+}
+
+function onDeleteRow(e) {
+  if (!e.target.classList.contains('deleteBtn')) {
+    return;
+  }
+
+  const btn = e.target;
+  btn.closest('section').remove();
+}
```

Comment:
- [ ] Regardless of the fact that your `onAddElement()` function is not currently working and therefore, I have been unable to test this function, it is quite obvious that you are manipulating the DOM to remove the book element without removing the actual book from the `books` array and the `localStorage`. Please factor this into your solution 👍